.changeset/brown-actors-enjoy.md

```diff
@@ -0,0 +1,12 @@
+---
+"@comet/cms-admin": patch
+---
+
+Fix styles of `ContentScopeSelect`
+
+-   Add Domain icon to list items
+-   Fix spacings
+-   Fix color of sub headers
+-   Fix size of `ClearInputAdornment` icon
+-   Fix mobile width and height to fill whole screen
+-   Fix `fontWeight` of selected list item
```

Comment:
I'm not sure this level of detail is really needed... I think something like this would be enough:

```suggestion
Adapt styling of `ContentScopeSelect` to match the Comet design
```